<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SOFUS (SIMD Optimized Fast Ultrasound Simulator)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_xdc">
       <attribute name="title">
        <string>Transducer</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QGroupBox" name="groupBox">
               <property name="toolTip">
                <string comment="Adjective">Type</string>
               </property>
               <property name="title">
                <string>Type</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_3">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_3">
                    <item>
                     <widget class="QLabel" name="label">
                      <property name="text">
                       <string>S&amp;hape</string>
                      </property>
                      <property name="buddy">
                       <cstring>cboxShape</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="cboxShape">
                      <item>
                       <property name="text">
                        <string>Linear</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Curvelinear</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_5">
                    <item>
                     <widget class="QLabel" name="label_2">
                      <property name="text">
                       <string>B&amp;affle</string>
                      </property>
                      <property name="buddy">
                       <cstring>cboxBaffle</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QComboBox" name="cboxBaffle">
                      <item>
                       <property name="text">
                        <string>Hard</string>
                       </property>
                      </item>
                      <item>
                       <property name="text">
                        <string>Soft</string>
                       </property>
                      </item>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_8">
               <property name="title">
                <string>Geometry</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_12">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_20">
                  <item>
                   <widget class="QLabel" name="label_16">
                    <property name="text">
                     <string>Elevation arc placement</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cboxElePlacement">
                    <item>
                     <property name="text">
                      <string>Inner</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Outer</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_2">
               <property name="title">
                <string>Elements</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_9">
                  <item>
                   <widget class="QTableView" name="tblElements">
                    <property name="minimumSize">
                     <size>
                      <width>5</width>
                      <height>2</height>
                     </size>
                    </property>
                    <attribute name="horizontalHeaderCascadingSectionResizes">
                     <bool>false</bool>
                    </attribute>
                    <attribute name="verticalHeaderCascadingSectionResizes">
                     <bool>false</bool>
                    </attribute>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="mplwindow" native="true">
          <property name="minimumSize">
           <size>
            <width>325</width>
            <height>245</height>
           </size>
          </property>
          <property name="baseSize">
           <size>
            <width>325</width>
            <height>245</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="mplvl"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_field">
       <attribute name="title">
        <string>Field</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QWidget" name="widget_3" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_14">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QGroupBox" name="groupBox_3">
               <property name="title">
                <string>Spatial Response</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <item>
                   <widget class="QLabel" name="label_3">
                    <property name="text">
                     <string>P&amp;ropagator</string>
                    </property>
                    <property name="buddy">
                     <cstring>cboxPropagator</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                    <widget class="QComboBox" name="cboxPropagator">
                     <item>
                      <property name="text">
                       <string>Point</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Plane</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>Sphere</string>
                      </property>
                     </item>
		    </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_9">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="text">
                     <string>I&amp;ntegration order</string>
                    </property>
                    <property name="buddy">
                     <cstring>sboxOrder</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="sboxOrder"/>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_10">
                  <item>
                   <widget class="QLabel" name="label_5">
                    <property name="text">
                     <string>D&amp;omain</string>
                    </property>
                    <property name="buddy">
                     <cstring>cboxDomain</cstring>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="cboxDomain">
                    <item>
                     <property name="text">
                      <string>Continous wave</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>Pulsed wave</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_17">
               <item>
                <widget class="QGroupBox" name="groupBox_4">
                 <property name="title">
                  <string>Excitation</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_11">
                    <item>
                     <widget class="QLabel" name="label_6">
                      <property name="text">
                       <string>C&amp;enter frq.</string>
                      </property>
                      <property name="buddy">
                       <cstring>leExcitationFrq</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leExcitationFrq">
                      <property name="text">
                       <string>7000000</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_10">
                      <property name="text">
                       <string>Hz</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_12">
                    <item>
                     <widget class="QLabel" name="label_8">
                      <property name="text">
                       <string>S&amp;ample frq.</string>
                      </property>
                      <property name="buddy">
                       <cstring>leSampleFrq</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leSampleFrq">
                      <property name="text">
                       <string>50000000</string>
                      </property>
		    </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_11">
                      <property name="text">
                       <string>Hz</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_13">
                    <item>
                     <widget class="QLabel" name="label_7">
                      <property name="text">
                       <string># of &amp;cycles</string>
                      </property>
                      <property name="buddy">
                       <cstring>leExcitationCycles</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leExcitationCycles">
                      <property name="text">
                       <string>2</string>
                      </property>
		     </widget>
                     </item>
                   </layout>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QGroupBox" name="groupBox_7">
                 <property name="title">
                  <string>Focus</string>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_11">
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_18">
                    <item>
                     <widget class="QLabel" name="label_12">
                      <property name="text">
                       <string>Focus &amp;distance</string>
                      </property>
                      <property name="buddy">
                       <cstring>leFocusDist</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leFocusDist">
                      <property name="text">
                       <string>0.02</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_13">
                      <property name="text">
                       <string>m</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <layout class="QHBoxLayout" name="horizontalLayout_19">
                    <item>
                     <widget class="QLabel" name="label_15">
                      <property name="text">
                       <string>&amp;F-number</string>
                      </property>
                      <property name="buddy">
                       <cstring>leFNumber</cstring>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLineEdit" name="leFNumber">
                      <property name="text">
                       <string>2</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QLabel" name="label_14">
                      <property name="text">
                       <string/>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                  <item>
                   <spacer name="verticalSpacer_3">
                    <property name="orientation">
                     <enum>Qt::Vertical</enum>
                    </property>
                    <property name="sizeHint" stdset="0">
                     <size>
                      <width>20</width>
                      <height>40</height>
                     </size>
                    </property>
                   </spacer>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_5">
               <property name="title">
                <string>Grid</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <widget class="QTableView" name="tblGrid">
                  <property name="minimumSize">
                   <size>
                    <width>4</width>
                    <height>6</height>
                   </size>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QGroupBox" name="groupBox_6">
               <property name="title">
                <string>Attenuation</string>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_10">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_16">
                  <item>
                   <widget class="QCheckBox" name="checkAttenuation">
                    <property name="text">
                     <string>Enable</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="leAttenuation">
                    <property name="text">
                     <string>0.54</string>
		    </property> 
		   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_9">
                    <property name="text">
                     <string>dB/(cm MHz)</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QWidget" name="mplwindow1" native="true">
             <property name="minimumSize">
              <size>
               <width>325</width>
               <height>245</height>
              </size>
             </property>
             <layout class="QVBoxLayout" name="mplvl1"/>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_15">
             <item>
              <widget class="QPushButton" name="btnField">
               <property name="text">
                <string>Calc</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QProgressBar" name="progressBar">
               <property name="value">
                <number>24</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cboxShape</tabstop>
  <tabstop>cboxBaffle</tabstop>
  <tabstop>tblElements</tabstop>
  <tabstop>cboxPropagator</tabstop>
  <tabstop>sboxOrder</tabstop>
  <tabstop>cboxDomain</tabstop>
  <tabstop>leExcitationFrq</tabstop>
  <tabstop>leSampleFrq</tabstop>
  <tabstop>leExcitationCycles</tabstop>
  <tabstop>leFocusDist</tabstop>
  <tabstop>leFNumber</tabstop>
  <tabstop>tblGrid</tabstop>
  <tabstop>checkAttenuation</tabstop>
  <tabstop>leAttenuation</tabstop>
  <tabstop>btnField</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
